apiserver/action/action.go

```diff
@@ -229,7 +229,7 @@ func (a *ActionAPI) ListCompleted(arg params.Entities) (params.ActionsByReceiver
 }
 
 // Cancel attempts to cancel enqueued Actions from running.
-func (a *ActionAPI) Cancel(arg params.Entities) (params.ActionResults, error) {
+func (a *ActionAPI) Cancel(arg params.Actions) (params.ActionResults, error) {
```

Comment:
What prompted this change? This breaks compatibility. The format of Entities is:

`{"entities": [{"tag":"..."}]}`

whereas the format of Actions is:

`{"actions": [{"tag":"...", ...}]}`

Also, only the tag is relevant, so Entities is more appropriate?